src/test/java/com/thomashan/coup/action/ActionValidatorUtilTest.java

```diff
@@ -1,7 +1,8 @@
-package com.thomashan.coup.action.state;
+package com.thomashan.coup.action;
 
 import com.thomashan.coup.action.Action;
 import com.thomashan.coup.action.ActionType;
+import com.thomashan.coup.action.ActionValidatorUtil;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No need to import a type that lives in the same package](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4558920)